target_chains/starknet/contracts/src/pyth.cairo

```diff
@@ -66,6 +66,7 @@ pub enum UpdatePriceFeedsError {
     Wormhole: super::wormhole::ParseAndVerifyVmError,
     InvalidUpdateData,
     InvalidUpdateDataSource,
+    InsufficientFee,
```

Comment:
Maybe `InsufficientFeeAllowance` can make this more clear.

Reply:
Renamed.